tracer/src/Datadog.Trace/Activity/DiagnosticSourceEventListener.cs

```diff
@@ -86,7 +86,10 @@ internal static void OnNextActivity<T>(string sourceName, KeyValuePair<string, o
                     return;
                 }
 
-                if (activity.Instance != null && activity.OperationName.Length != dotIndex && string.Compare(activity.OperationName, 0, value.Key, 0, dotIndex, StringComparison.Ordinal) != 0)
+                if (activity.Instance != null
```

Comment:
Super nit-picky, but you're using `!= null` in one line and `is not null` in the next. Pick one and be consistent, please! 😅

Reply:
Done in 167db7f98af92392b0cb783cb8a36d0477cfe47d

Reply:
yay!